### DISCUSSION TITLE
Rectangle Pro to manage Sonoma widget placement?

### DISCUSSION THREAD

**Discussion by Coder84619** [rxhanson/RectanglePro-Community#360] [Ideas]
Are there macOS APIs that would allow Rectangle Pro to move Sonoma desktop widgets? I have two different monitor configurations (home/work), and having widgets placed properly would be great.

**Comment by rxhanson** [comment]
Thanks for the feedback. As far as I know, there is no public API for Sonoma widgets, but I'll have to investigate more and see what I can do.
